### PR TITLE
num_workers=0 works in windows, potentially a bug in pytorch 

### PR DESCRIPTION
**Pull request by danzz006**
Got it working by setting num_workers=0 in dataloader (seems like it does not work correctly even with one worker in windows), still bugs in training. **loss** updates but **accuracy** remains 0. Need to check whole loop again

### CODE REVIEW

**Comment by Rijha** [comment]
AoA
Training accuracy is only 58%.
losses are very high and validation accuracy is very low. I guess data set
contains less samples. By increasing batch size performance improves but
can't increase batch size more than 5.cuda memory issue.
Attached  is the link of notebook file.
https://github.com/Rijha/AVD/blob/main/AutomaticVulnerablilityDetectionwithBertwith2classes.ipynb

Regards,
Rijha
